projects/igniteui-angular/src/lib/dialog/dialog.component.ts

```diff
@@ -147,6 +307,16 @@ export class IgxDialogComponent implements IToggleView, OnInit, OnDestroy {
         }
     }
 
+    /**
+     *An accessor that returns the value of the `_titleId` property.
+     *```html
+     *@ViewChild("MyDialog")
+     *public dialog: IgxDialogComponent;
+     *ngAfterViewInit() {
+     *    let dialogOpen = this.dialog.titleId;
```

Comment:
Maybe change the variable name to make more sense.

Reply:
Will do.